### PR TITLE
Update wesnoth-ala.po

### PR DESCRIPTION
**Pull request by MultiSeptim**
Cooljeanius, please wait for @celerini to come up with suggestions. I will let you know when it is ready for merge.

### CODE REVIEW

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 1]
```diff
@@ -2678,6 +2683,10 @@ msgid ""
 "not only the defense of the town, but also its economic rebirth, committing "
 "itself to reopening the legendary Amber Road! "
 msgstr ""
+"Mladý důstojník wesnothské armády je poslán, aby velel malé posádce v "
+"bezvýznamném městečku v divočině dálného východu. Nečekaně zvládne nejen "
+"obranu města, ale i jeho ekonomické znovuzrození, zavazujíc se tak "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]: masculine transgressive
```suggestion
"obranu města, ale i jeho ekonomické znovuzrození, zavazuje se tak "
```

**Review by celerini — CHANGES_REQUESTED** [review]

File: translations/cs/LC_MESSAGES/wesnoth-ala.po (hunk 1 above)
Comment (cooljeanius) [reply translations/cs/LC_MESSAGES/wesnoth-ala.po]: hm, let's see where the commit will go if I click the "Commit suggestion" button... to your branch, or mine?

Comment (cooljeanius) [reply translations/cs/LC_MESSAGES/wesnoth-ala.po]: ok it looks like it went to Septim's branch

**Comment by cooljeanius** [comment]
sorry I introduced a merge conflict...

**Pull request merged**